nifty-script

```diff
@@ -367,15 +367,8 @@ travis_for_separated_site_repository() {
 
     __ensure_reqs_and_decrypt
 
-    # Note: The -vv below is important for correctness. We have some
```

Comment:
I know it's merged, and it looks good for what it is, but did we address the issue laid out in the comment?